spec.html

```diff
@@ -12123,7 +12123,7 @@ <h1>Interesting Cases of Automatic Semicolon Insertion</h1>
 
     <p>As new syntactic features are added to ECMAScript, additional grammar productions could be added that cause lines relying on automatic semicolon insertion preceding them to change grammar productions when parsed.</p>
 
-    <p>The interesting cases of automatic semicolon insertion are places where a semicolon may or may not be inserted depending on preceding source text according to the rules above. These places are considered interesting if changes outside of an existing source text grammar production could change the grammar production of the existing source text depending on automatic semicolon insertion. The rest of this section describes a number of interesting cases of automatic semicolon insertion in this version of ECMAScript.</p>
+    <p>The interesting cases of automatic semicolon insertion are thus places where a semicolon may or may not be inserted, depending on the source text which precedes it. The rest of this section describes a number of interesting cases of automatic semicolon insertion in this version of ECMAScript.</p>
```

Comment:
The previous wording was effectively defining "interesting", which I would prefer to keep. For example:
```suggestion
    <p>For the purposes of this section a case of automatic semicolon insertion is considered interesting if it is a place where a semicolon may or may not be inserted, depending on the source text which precedes it. The rest of this section describes a number of interesting cases of automatic semicolon insertion in this version of ECMAScript.</p>
```

Reply:
+1 to this suggestion.

Reply:
I was hoping the single word "thus" sufficed to convey that, but more explicit is good. :smile:

Reply:
Added.